content/docs/04.5-devx/05-app-profile/10-service-listings/2-postgresql-db.md

```diff
@@ -0,0 +1,132 @@
+|Application Profile Name | A custom name for the App Profile|
+|Version (optional)       | The default value is 1.0.0. You can create multiple versions of an App Profile using the format **`major.minor.patch`**.
+|Description (optional)   | Description of the App Profile. | 
+|Tag (optional)           | Tags on a cluster group are propagated to the cloud/datacenter environments.|
+ 
+
+4. From the available services start configuring your App Profile. Refer to [App Profiles](/devx/app-profile) for a list of available services.
+
+
+5. Click on **PostgreSQL** from the DB services and start the configuration.
+  
+
+6. Provide the following information to the wizard:
+
+  * **Name:** The database *service name*. You can have the default Palette generated name or create a custom name.
+
+
+  * **Username:** The user name for DB access control. 
+
+
+  * **Password:** Security password for the DB service.
+
+You can use the default system-generated password. If the default password is used, it can be retrieved from the PostgreSQL secrets using the following command:
+
+<br />
+
+```
+kubectl get secrets -A
+```
+<br />
+
+For using a custom password, use the [base 64 encoder](https://www.base64encode.org/) to generate an encoded password and add to the basic information wizard. 
+
+  * **Database Volume Size (GiB):** Select the volume as per the storage volume available in the cluster group and virtual clusters. 
+
+  * **Version:**Select the version from the **Version** drop-down. The following are the Palette supported PostgreSQL versions:
+
+    * 14
+   
+
+6. **Output Variables**: The output variables of this service layer that may be used in higher service layers, typically for connection purposes are:
+
+  * The database service name must be passed on to the output variables for database connectivity to other app services. 
+
+  * Set `sslMode` to `require` to [establish database connectivity](https://www.postgresql.org/docs/current/libpq-ssl.html).
+
+
+```
+{{.spectro.app.$appDeploymentName.database-<service-name>.USERNAME}}
+```
+```
+{{.spectro.app.$appDeploymentName.database-<service-name>.PASSWORD}}
+```
+```
+{{.spectro.app.$appDeploymentName.database-<service-name>.POSTGRESMSTR_SVC}}
+```
+```
+{{.spectro.app.$appDeploymentName.database-<service-name>.POSTGRESMSTR_SVC_PORT}}
+```
+
+|**Output Variable**|**Description**|
+|---------------|-----------|
+|Username|Username for database access control|
+|Password|Password for database access control|
+|POSTGRESMSTR_SVC|Provides the Postgres service fully qualified domain name (FQDN) which can be consumed by App Services for database connectivity|
+|POSTGRESMSTR_SVC_PORT|Represents the port on which the database service is listening to|
+
+
+# Validation
+
+* To validate your database service in App Profile, navigate to the **App Profiles** page, where all your app profiles are listed. Click the **App Profile Name** to see the service layers.
```

Comment:
In line 116: Select the app profile and review the service layers. The table lists the different service layers that make up the app profile.

Sight rewording, missing word, and there's a table instead of a screen.